### ISSUE TITLE
Unable to initialize project using cli

### ISSUE THREAD

**Issue by morfert** [edgedb/edgedb-cli#1036]
- EdgeDB Version: 2.14+7aec755
- EdgeDB CLI Version: 3.0.0-dev.983+bbf07e5
- OS Version: Windows 10 Home 19044.2965

Steps to Reproduce:

1. Install edgedb on windows with wsl2
2. Run 'edgedb project init'

I referenced https://github.com/edgedb/edgedb-cli/issues/850#issuecomment-1275965281_ from #850 by @tailhook , installed nightly and ran 'edgedb project init 2>project-init.log' to generate the following log.
[project-init.log](https://github.com/edgedb/edgedb-cli/files/11468921/project-init.log)

**Comment by tailhook** [comment]
The only thing we see in the log is that network in WSL doesn't work. It's likely that firewall [blocks the network in WSL](https://github.com/microsoft/WSL/issues/4139). Unfortunately there are a lot of different issues with WSL and firewalls.

If you don't have any specific one installed I suggest trying to [disable "windows defender firewall"](https://support.microsoft.com/en-us/windows/turn-microsoft-defender-firewall-on-or-off-ec0844f7-aebd-0583-67fe-601ecf5d774f) entirely and test if that works. If it does you can revert the setting and work thought more fine-grained firewall tuning.

**Comment by morfert** [comment]
Yeah, I only have windows firewall installed. Even after disabling all the firewall options from settings I could not initialize a project.
